controllers/taloscontrolplane_controller.go

```diff
@@ -709,17 +701,16 @@ func (r *TalosControlPlaneReconciler) reconcileKubeconfig(ctx context.Context, c
 		)
 		if createErr != nil {
 			if createErr == kubeconfig.ErrDependentCertificateNotFound {
-				return errors.Wrapf(&capierrors.RequeueAfterError{RequeueAfter: 30 * time.Second},
-					"could not find secret %q for Cluster %q in namespace %q, requeuing",
-					secret.ClusterCA, clusterName.Name, clusterName.Namespace)
+				r.Log.Info("could not find secret %q for Cluster %q in namespace %q, requeuing", secret.ClusterCA, clusterName.Name, clusterName.Namespace)
```

Comment:
Is it necessary to log this? And if so, would it be on `Info` level or on `Error` level?